<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactsItem</class>
 <widget class="QWidget" name="ContactsItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>31</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QLabel" name="user_status">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>:/icons/16/globe</pixmap>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="user_name">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="user_actions">
     <property name="text">
      <string>actions</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="delete_user">
     <property name="text">
      <string>delete</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="view_profile">
     <property name="toolTip">
      <string>See this persons profile</string>
     </property>
     <property name="text">
      <string>profile</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="send_message">
     <property name="toolTip">
      <string>Send message to</string>
     </property>
     <property name="text">
      <string>message</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="share_file">
     <property name="toolTip">
      <string>Send file(s) to </string>
     </property>
     <property name="text">
      <string>share</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
